tutorials/setup_tutorial.md

```diff
@@ -0,0 +1,255 @@
+* `target_tasks`: This is a comma-delimited string of tasks you want to fine-tune and evaluate on (in this case "sts-b,wnli").
+* `word_embs`: This is a string specifying the type of word embedding you want to use. In `config/demo.conf`, this is already set to `scratch`. *
+* `val_interval`: This is the interval (in steps) at which you want to evaluate your model on the validation set during pretraining. A step is a batch update.
+* `exp_name`, which expects a string of your experiment name. 
+* `run_name`, which expects a string of your run name.
+The main differences between an experiment and a run is that all runs in an experiment will have the same preprocessing (which may include tokenization differences), while runs his a specific experiment (which may differ from other runs by learning rate or type of sentence encoder, for example, your run name could be `rnn_encoder` for a run with a biLSTM sentence encoder).
+
+Additionally, let's suppose we want to have parameters specific to the STS-B task. Then, we can simply specify them as here:
+
+```python
+
+sts-b += {
+    classifier_hid_dim = 512
+    pair_attn = 0
+    max_vals = 16
+    val_interval = 10  
+}
+
+```
+
+Your finished product should look like the below:
+
+```
+include "defaults.conf"  // relative path to this file
+
+// write to local storage by default for this demo
+exp_name = jiant-demo
+run_name = mtl-sst-mrpc
+
+cuda = 0
+random_seed = 42
+
+load_model = 0
+reload_tasks = 0
+reload_indexing = 0
+reload_vocab = 0
+
+pretrain_tasks = "sst,mrpc"
+target_tasks = "sts-b,wnli"
+classifier = mlp
+classifier_hid_dim = 32
+max_seq_len = 10
+max_word_v_size = 1000
+
+word_embs = scratch
+d_word = 50
+
+sent_enc = rnn
+skip_embs = 0
+
+batch_size = 8
+
+val_interval = 50
+max_vals = 10
+target_train_val_interval = 10
+target_train_max_vals = 10
+
+// Use += to inherit from any previously-defined task tuples.
+sts-b += {
+    classifier_hid_dim = 512
+    pair_attn = 0
+    max_vals = 16
+    val_interval = 10  
+}
+
+```
+
+Now we get on to the actual experiment running!
+To run the experiment, you can simply run the below command via command line, or use a bash script. 
+
+ You can use the `--overrides` flag to override specific variables. For example:
+
+```sh
+python main.py --config_file config/tutorial.conf \
+    --overrides "exp_name = my_exp, run_name = foobar"
+```
+
+will run the demo config, but write output to `$JIANT_PROJECT_PREFIX/my_exp/foobar`.
+
+
+### 3.b) Understanding the output logs
+
+We do support Tensorboard, however, you can also look at the logs to make sure everything in your experiment is running smoothly. 
+
+The logs include:
```

Comment:
"There's a lot going on here (including some debugging information that we're working on suppressing), but a lot of it is useful:"